dotenv-sample

```diff
@@ -1,6 +1,13 @@
-# See comments in ebmbot/settings.py
+# See comments in ebmbot/settings.py and DEVELPOPERS.md
+# For local dev environments, these variables can be
+# configured by running ./scripts/local-setup.sh
 LOGS_DIR=changeme
-WRITEABLE_DIR=.
+WRITEABLE_DIR=changeme
+GCP_CREDENTIALS_PATH=changeme/gcp-credentials.json
+# Use these settings for running in docker
+# GCP_CREDENTIALS_PATH="/app/writeable_dir/gcp-credentials.json"
+# LOGS_DIR="/app/logs"
+# WRITEABLE_DIR="/app/writeable_dir"
```

Comment:
Very pedantic but mayb ekeep the order the same?
```suggestion
# LOGS_DIR="/app/logs"
# WRITEABLE_DIR="/app/writeable_dir"
# GCP_CREDENTIALS_PATH="/app/writeable_dir/gcp-credentials.json"
```

Reply:
Oh good point, will fix :+1: